changelog.mdx

```diff
@@ -5,6 +5,43 @@ metaDescription: Chalk's latest updates.
 description: Updates to Chalk!
 ---
 
+## December 23, 2024
+
+### Search and filter logs by query name, log message, deployment, and more!
+
+You can now search logs in the Logs Explorer in the dashboard using a key-value pair query format:
+`key1:value1 key2:value2 ...`. This allows you to combine multiple filters at once. The Logs Explorer
+displays the available keys for filtering, shown below.
+
+![Updated log explorer](/img/log-search-filters.png)
+
+### Search by file name in the source code viewer
+
+In the Source Code viewer, found in the Deployments page of the dashboard, you can now search for files by name
+in addition to the existing text search through the source code from the linked deployment.
```

Comment:
```suggestion
We've extended our Dashboard's Deployment Source Code Viewer! In addition to searching
for keywords in your source code, you can now also search for your files by name.
```
Maybe a little clearer but don't feel super strongly about this suggestion—feel free to ignore :)